scripts/utils.py

```diff
@@ -0,0 +1,32 @@
+# This script contains functionality to simplify compose scripts for new datasets
+
+import pandas as pd
+
+# All ID's we use to search in OpenAlex
+ID_SET = ["doi", "pmid", "title", "year"]
```

Comment:
By making this into an actual Python set, you can simplify the code below with an intersection I think.